draft-ietf-tls-dtls-rrc.md

```diff
@@ -160,6 +174,50 @@ threshold) and initiate the return routability check that proceeds as follows:
 After this point, any pending send operation is resumed to the bound peer
 address.
 
+{{path-challenge-reqs}} and {{path-response-reqs}} contain the requirments for
+the initiator and responder roles, broken down per protocol phase.
+
+## Path Challenge Requirements {#path-challenge-reqs}
+
+* The initiator MAY send multiple `return_routability_check` messages of type
+  path_challenge to cater for packet loss on the probed path.
+  * Each path_challenge SHOULD go into different transport packets.
+  * Each path_challenge MUST have different unpredictable data.
+* The initiator MAY use padding using the record padding mechanism available in
+  DTLS 1.3 (and in DTLS 1.2, when CID is enabled on its sending direction) up to the
+  anti-amplification limit to probe if the PMTU for the new path is still
+  acceptable.
+
+## Path Response Requirements {#path-response-reqs}
+
+* The responder MUST NOT delay sending an elicited path_response message.
+* The responder MUST send exactly one path_response messages for each received
+  path_request.
+* The responder MUST send the path_response on the network path where the
+  corresponding path_challenge has been received, so that validation succeeds
+  only if the path is functional in both directions.
+  * The initiator MUST NOT enforce this behaviour
+* The responder MAY use padding using the record padding mechanism available in
+  DTLS 1.3 (and in DTLS 1.2, when CID is enabled on its sending direction) up to the
+  anti-amplification limit to probe if the PMTU for the new path is still
+  acceptable.
```

Comment:
Remove this.  Replace with text that says that if responder wants to do PMTU discovery it can run another RRC protocol instance with itself as the initiator.

Reply:
addressed in df82fd8